tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["DOM", "ES2022", "DOM.Iterable"],
     "jsx": "react",
     "module": "Node16",
-    "moduleResolution": "Node",
```

Comment:
Node is an invalid moduleResolution when module is set to Node16. The CJS build config reverts this since it uses CJS module format.